requirements/dev-requirements.txt

```diff
@@ -6,108 +6,108 @@
+click==7.1.2              # via git-build-branch, pip-tools
+cloudant==2.14.0          # via jsonobject-couchdbkit
+colorama==0.4.3           # via sniffer
 commcaretranslationchecker==0.9.7  # via -r requirements/requirements.in
 commonmark==0.9.1         # via recommonmark
 concurrent-log-handler==0.9.12  # via -r requirements/requirements.in
-contextlib2==0.5.5        # via git-build-branch, schema
+contextlib2==0.6.0.post1  # via git-build-branch, schema
 coverage==4.5.1           # via -r requirements/test-requirements.in
 csiphash==0.0.5           # via -r requirements/requirements.in
-cssselect2==0.2.2         # via cairosvg, weasyprint
+cssselect2==0.3.0         # via cairosvg, weasyprint
 datadog==0.15.0           # via -r requirements/requirements.in
 ddtrace==0.14.1           # via -r requirements/requirements.in
 decorator==4.0.11         # via -r requirements/requirements.in, datadog, ipython, jsonpath-rw, traitlets
 defusedxml==0.5.0         # via -r requirements/requirements.in, cairosvg
 diff-match-patch==20120106  # via -r requirements/requirements.in
 dimagi-memoized==1.1.3    # via -r requirements/requirements.in
 django-angular==2.2.4     # via -r requirements/requirements.in
-django-appconf==1.0.3     # via django-compressor, django-statici18n
+django-appconf==1.0.4     # via django-compressor, django-statici18n
 django-braces==1.13.0     # via -r requirements/requirements.in
 django-bulk-update==2.2.0  # via -r requirements/requirements.in
 django-celery-results==1.0.4  # via -r requirements/requirements.in
 django-compressor==2.1    # via -r requirements/requirements.in
 django-countries==4.6     # via -r requirements/requirements.in
 django-crispy-forms==1.7.0  # via -r requirements/requirements.in
 django-cte==1.1.4         # via -r requirements/requirements.in
-django-debug-toolbar==2.0  # via -r requirements/dev-requirements.in
-django-extensions==2.2.5  # via -r requirements/dev-requirements.in
+django-debug-toolbar==2.2  # via -r requirements/dev-requirements.in
+django-extensions==3.0.8  # via -r requirements/dev-requirements.in
 django-formtools==2.1     # via -r requirements/requirements.in, django-two-factor-auth
 django-logentry-admin==1.0.6  # via -r requirements/requirements.in
 git+git://github.com/dimagi/django-nose@fast-first-1.4.6#egg=django-nose  # via -r requirements/test-requirements.in
 django-oauth-toolkit==1.3.2  # via -r requirements/requirements.in
-django-otp==0.9.3         # via -r requirements/requirements.in, django-two-factor-auth
+django-otp==0.9.4         # via -r requirements/requirements.in, django-two-factor-auth
 django-partial-index==0.6.0  # via -r requirements/requirements.in
-django-phonenumber-field==1.3.0  # via django-two-factor-auth
-django-prbac==0.0.8       # via -r requirements/requirements.in
+django-phonenumber-field==2.3.1  # via -r requirements/requirements.in, django-two-factor-auth
+django-prbac==1.0.1       # via -r requirements/requirements.in
```

Comment:
Would be nice to create a release in the github repo with the changes since 0.0.8: https://github.com/dimagi/django-prbac/compare/0.0.8...b7d312474f7703ae8521c18ed7481156ea58111c

Reply:
Thanks, I had forgotten to add the tag on github. https://github.com/dimagi/django-prbac/releases/tag/1.0.1